sdk/storage/Azure.Storage.Blobs/src/Sas/BlobSasBuilder.cs

```diff
@@ -27,8 +27,9 @@ public struct BlobSasBuilder : IEquatable<BlobSasBuilder>
         /// <summary>
         /// The optional signed protocol field specifies the protocol
         /// permitted for a request made with the SAS.  Possible values are
```

Comment:
Is there a scenario in the future where they might be more options available that we will need to update this comment every time?

Reply:
Yes, I'm sure some of these will need to be updated.  I think that's probably an okay tax on our team to provide a better doc experience for users?

Reply:
In general, parameters to SAS do get expanded over time.  Sometimes rapidly.